.github/workflows/dependency-review.yaml

```
@@ -10,3 +10,5 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+    with:
+      base-ref: ${{ github.event.pull_request.base.sha }} || 'master'
```

```suggestion
      base-ref: ${{ github.event.pull_request.base.sha || 'master' }}
```

I'm just tired today. Sorry.